src/components/learner-credit-management/data/utils.js

```diff
@@ -570,6 +570,29 @@ export const minimumEnrollByDateFromToday = ({ subsidyExpirationDatetime }) => M
   dayjs(subsidyExpirationDatetime).subtract(MAX_ALLOWABLE_REFUND_THRESHOLD_DAYS, 'days').toDate(),
 );
 
+const subsidyExpirationRefundCutoffDate = ({ subsidyExpirationDatetime }) => dayjs(subsidyExpirationDatetime).subtract(MAX_ALLOWABLE_REFUND_THRESHOLD_DAYS, 'days').toDate();
+
+const isStartDateWithinThreshold = ({ start, enrollStart, subsidyExpirationDatetime }) => {
+  if (!start && !enrollStart) {
+    return true;
+  }
+  const timeStampStartDate = dayjs(start).unix();
+  const timeStampEnrollStartDate = dayjs(enrollStart).unix();
+  const earliestStartDate = Math.min(timeStampStartDate, timeStampEnrollStartDate);
+  return dayjs(
+    earliestStartDate,
+  ).isBefore(subsidyExpirationRefundCutoffDate({ subsidyExpirationDatetime }), 'seconds');
+};
+
+const isEnrollByDateWithinThreshold = ({ enrollBy, isLateRedemptionAllowed = false }) => {
+  if (!enrollBy) { return true; }
+  let enrollByCutoffDate = dayjs();
```

Comment:
nit: this might read more naturally as `enrollmentEffectiveDate`.
Then the return statement would read as `return dayjs(enrollBy).isAfter(enrollmentEffectiveDate, 'seconds');`, which (IMHO) is slightly easier to parse the intention of.

Reply:
Updated 👍🏽